package.json

```diff
@@ -4,7 +4,8 @@
   "version": "1.0.0",
   "author": "The Tab for a Cause Team",
   "dependencies": {
-    "@fullpage/react-fullpage": "^0.1.18",
+    "@emotion/react": "^11.4.1",
```

Comment:
Are we using Emotion?

Reply:
one of our other dependencies required it as a peer

Reply:
I've moved it to a dev dependency

Reply:
@amaliwanag2 As discussed, stick this back in dependencies, then LGTM!